content/_data/changelogs/weekly.yml

```diff
@@ -10070,6 +10070,57 @@
   # pull: 5183 (PR title: Ignore old servlet api in dependabot)
   # pull: 5185 (PR title: Bump Mailer from 1.23 to 1.32.1)
 
+- version: '2.277'
+  date: 2021-01-26
+  changes:
+  - type: rfe
+    category: rfe
+    pull: 5176
+    authors:
+    - fqueiruga
+    - timja
+    message: |-
+      Use a more accessible color palette in configuration form tabs.
+  - type: rfe
+    category: rfe
+    pull: 5198
+    authors:
+    - res0nance
+    message: |-
+      Improve performance when saving fingerprints to disk.
+  - type: rfe
+    category: rfe
+    pull: 5190
+    issue: 64670
+    authors:
+    - alecharp
+    - batmat
+    message: |-
+      Improve performance in fingerprint file creation.
+  - type: rfe
+    category: rfe
+    pull: 5177
+    issue: 64291
+    authors:
+    - fqueiruga
+    message: |-
+      Replace the drag & drop library used by Jenkins with SortableJS.
```

Comment:
Move this sentence to internal or remove entirely? User-facing behavior is more important IMHO

Reply:
See aa3e71db1